GDevelop.js/package.json

```diff
@@ -13,8 +13,8 @@
     "port"
   ],
   "scripts": {
-    "build": "grunt build",
-    "build-with-MinGW": "grunt build --use-MinGW",
+    "build": "node scripts/sync-versions.js && grunt build",
+    "build-with-MinGW": "node scripts/sync-versions.js && grunt build --use-MinGW",
```

Comment:
Instead of doing this in the package.json (which I think won't work on Windows because of &&), let's do it in the gruntfile.js. It's a bit old fashioned, but should not be too complex:


Add a new task at the beginning of "build" tasks list:
<img width="373" alt="image" src="https://github.com/user-attachments/assets/d970e7a7-7877-4f43-9e07-a7bb763f416d">

and declare a task that runs your script:
<img width="476" alt="image" src="https://github.com/user-attachments/assets/ef5fcbcd-7fd5-430d-9af3-c232a4be7391">

It's a tiny bit more boilerplate, but at least we're keeping things in the gruntfile.js (which we will probably one day rewrite as... a pure, simple node.js file, and remove grunt entirely - it's useless)